hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/client/api/impl/TimelineV2ClientImpl.java

```diff
@@ -34,9 +34,9 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.CancellationException;
 
-import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.MultivaluedMap;
```

Comment:
presumably we will want to cut this too, in future...

Reply:
Yes - I think so but I need to be more research about MultivaluedMap to work out what the alternative is.